input/aragog.toml

```diff
@@ -239,9 +239,10 @@ author  = "Harrison Nicholls, Tim Lichtenberg"
     # Set initial volatile inventory by planetary element abundances
     [delivery.elements]
         CH_ratio    = 1.0       # C/H ratio in mantle/atmosphere system
-        H_oceans    = 6.0       # Hydrogen inventory in units of equivalent Earth oceans, by mass
-        N_ppmw      = 2.0       # Nitrogen inventory in ppmw relative to mantle mass, by mass
-        S_ppmw      = 200.0     # Sulfur inventory in ppmw relative to mass of melt
+        H_oceans    = 0.0       # Hydrogen inventory in units of equivalent Earth oceans
```

Comment:
So both settings are possible then? Does this also translate to the CALLIOPE repository itself?

Reply:
If so you need to make clear to set only one or the other somehow, via comments or if conditions.

Reply:
Yes, this makes both of them possible. In theory the user could use both at once, for whatever reason, but it probably makes sense for them to only set one of them to be >0 at any one time. 

Could add a warning for this I suppose.

Reply:
This doesn't require any changes to CALLIOPE, because it converts the H_ppmw to an equivalent amount of H_oceans before the value is passed to CALLIOPE.

Reply:
How would setting both of them make sense? This gives two different H inventories, no?

Reply:
The current formulation sums them together, so the total hydrogen inventory is now:

H_total = H_oceans * ocean + H_ppmw * mantle.

This obviously reduces quite nicely if the user just sets one of the two parameters. And it allows them to set both (which will generate a warning) if they would want to do that too.